.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true
+}
```

Comment:
It's better to generally exclude .vscode folder and add it to gitignore (If you do not plan to share the settings with other programmers).

Reply:
Done, hidden.